contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -462,25 +458,27 @@ contract JBETHPaymentTerminalStore {
       revert CURRENCY_MISMATCH();
     }
 
-    // Convert the amount to wei.
-    // A currency of 0 should be interpreted as whatever the currency being withdrawn is.
-    withdrawnAmount = _currency == 0
-      ? _amount
-      : PRBMathUD60x18.div(_amount, prices.priceFor(_currency, JBCurrencies.ETH));
+    uint256 _newOverflowAllowanceOf = usedOverflowAllowanceOf[_projectId][
```

Comment:
Decided to calculate this once here since this value is used effectively twice later on

Reply:
maybe `_newUsedOverflowAllowanceOf`

Reply:
nice change